README.md

```diff
@@ -9,6 +9,12 @@
 
 ---
 
+## Installation
+
+- Clone the repository.
+- Navigate to the directory and install dependencies with `npm install`.
+- Now fire in `npm link` which creates a symlink and now `hdg` can be accessed globally.
+
```

Comment:
It is better to move this to `contributing.md`, create a new section within `README` pointing to the contributing doc.